torch/optim/adagrad.py

```diff
@@ -188,6 +220,7 @@ def step(self, closure=None):
         {_foreach_doc}
         {_maximize_doc}
         {_differentiable_doc}
+        {_fused_doc}
```

Comment:
this doc says something very contrary to what this PR does haha. I would just not use it here at all and write something custom.

In fact now that you've added CPU fused for Adam(W) and SGD, a good thing to do that we missed is to update the doc!